index.js

```diff
@@ -242,10 +242,12 @@ function stdin() {
  */
 
 function renderFile(path, rootPath) {
-  var re = /\.(?:pug|jade)$/;
+  var isPug = /\.(?:pug|jade)$/;
+  var isIgnored = /(\/_)|(^_)/;
```

Comment:
Oh sorry, I replied before refreshing the page and didn't see your comment, I'll check this out then, thanks!